ui/components/common/Validators.js

```diff
@@ -1,3 +1,4 @@
+const MAX_POSTGRES_PORT = 65536;
```

Comment:
It's not the maximum value for Postgres, it't the maximum value any port can have.